<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>master_screen</class>
 <widget class="QMainWindow" name="master_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>KalaYab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>جستجو</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_input">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="categories_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>دسته بندی ها:</string>
    </property>
   </widget>
   <widget class="QLabel" name="laptop_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/laptop.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/laptop.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mobile_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/mobile.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/mobile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cooking_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/cooking.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/cooking.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clothing_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/cloth.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/cloth.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="digital_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/digital.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/digital.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="candb_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/carandbike.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/carandbike.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/kalayab-logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resorses/res.qrc">:/newPrefix/kalayab-logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>709</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ورود و ثبت نام</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGuest">
    <property name="title">
     <string>Guest</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menuGuest"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>ورود به حساب</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>خروج از حساب کاربری</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resorses/res.qrc"/>
 </resources>
 <connections/>
</ui>
